src/main/java/org/jitsi/protocol/xmpp/util/MediaSSRCGroupMap.java

```diff
@@ -33,22 +34,22 @@
     /**
      * Map backend.
      */
-    private final Map<String, List<SSRCGroup>> groupMap;
+    private final ConcurrentHashMap<String, List<SSRCGroup>> groupMap;
```

Comment:
Both MediaSSRCGroupMap and MediaSSRCMap classes are thread unsafe just like the HashMap is. So they should be synchronised on a higher level.